lib/queuesender/logger.go

```diff
@@ -0,0 +1,20 @@
+package queuesender
```

Comment:
Question: What would you think if prefixLogger methods work with nil receiver? That is, calling Fatalf, Printf, etc. on a nil prefixLogger would be a no-op. 

I found that feature to be useful for my stuff since the client providing a logger is optional. Rather than code like:

if logger != nil {
  logger.Println(err)
}

I can just do

logger.Println(err)

I know doing that may not make sense for everything though.